src/Mod/Arch/importWebGL.py

```diff
@@ -396,7 +396,7 @@ def getHTMLTemplate():
                                 m.defines.USE_DASH = ""; // https://discourse.threejs.org/t/dashed-line2-material/10825
                             }
                         }
-                        if (guiparams.wiretype == 'None') {
+                        if (guiparams.wiretype is 'None') {
```

Comment:
Sure? This is JS code, not Python.

Reply:
:face_with_spiral_eyes: my bad!

Reply:
amended commit info to reflect revision. TY wwmayer!